lib/rust/ensogl/example/slider/src/lib.rs

```diff
@@ -92,4 +93,20 @@ fn init(app: &Application) {
     slider4.inner().frp.use_overflow_bounds(Bounds::new(-2.0, 3.0));
     slider4.inner().frp.set_caption(Some("Caption".to_string()));
     slider4.inner().set_track_color(color::Rgba::new(0.5, 0.70, 0.70, 1.0));
+
+    let world = &app.display;
+
+    let mut task_handle = Some(Box::new(profiling::start_task!("mcdbg_test_task")));
+    let mut i = 0;
+    world
+        .on_frame(move |_| {
+            if i == 100 {
+                DEBUG!("MCDBG i==3");
+                if let Some(t) = task_handle.take() {
+                    t.end();
+                }
+            }
+            i += 1;
+        })
+        .forget(); // FIXME(akavel): what this does?
```

Comment:
I don't think there is any need for it here, as we call `mem::forget(app)` in the entrypoint. I think it is kind of an artifact of refactoring.

Reply:
Thanks! Anyway, I was thinking to remove this block before opening a regular PR, as it was mostly to test the feature; do you think I should keep it? 🤔

Reply:
Well, I think it doesn't have much sense in the `slider` example. I'm not sure if there is a better place for it

Reply:
> I don't think there is any need for it here, as we call `mem::forget(app)` in the entrypoint. I think it is kind of an artifact of refactoring.

As I see, we don't forget `app` here, but the handle of `on_frame` callback. Dropped callback are unregistered AFAIR.

Reply:
Ah I see, I missed that `on_frame` actually returns a handle. I believe the usage of `.forget()` method is a bit confusing here - it is not clear what it is applied to while reading the code.

Reply:
I deleted this block. It was here only for local testing while there were no real uses of the profiling framework in the main IDE code. Now that they exist, this seems unneeded.